tee-worker/litentry/pallets/identity-management/src/lib.rs

```diff
@@ -184,8 +163,8 @@ pub mod pallet {
 			identity: Identity,
 		) -> DispatchResult {
 			T::ManageOrigin::ensure_origin(origin)?;
+			Self::maybe_create_id_graph(&who)?;
```

Comment:
how is it possible that we reached this point without id graph being created before?

Reply:
> how is it possible that we reached this point without id graph being created before?

You could deactivate the prime identity.
As written in the PR description, I didn't find a reason not to allow it.

Reply:
I see so this means id graph is created but prime identity is deactivated - thanks.

Reply:
Hmm but deactivating identity is nothing more than changing it status. 
It will still be present in `IDGraphs` map, am I right

Reply:
Exactly, but "status" only exists in the context of an IDGraph. So if you deactivates the (prime) identity without linking any other identities, you kind of have to create the IDGraph first